.sync.yml

```diff
@@ -1,3 +0,0 @@
----
```

Comment:
Is this really the right thing to do?

Reply:
a previous version of this PR did a cleanup of the old travis options. I updated the PR.